backend/tests/test_services.py

```
@@ -0,0 +1,57 @@
+
+    def test_load_config_default(self):
+        config = self.service.get_config()
+        self.assertEqual(config['min_liquidity'], 10000)
+        self.assertEqual(config['buy_amount_sol'], 0.01)
+
+    def test_update_config(self):
+        self.service.update_config({'min_liquidity': 20000})
+        self.assertEqual(self.service.get_config()['min_liquidity'], 20000)
+
+    @patch('asyncio.create_task')
+    def test_start_stop_trading(self, mock_create_task):
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_create_task`

(ARG002)